src/index.js

```diff
@@ -12,11 +13,13 @@ import parseUrl from 'url-parse';
   *
   * If you feel something is missing, not clear or could be improved, please don't hesitate to open an [issue in GitHub](https://github.com/agile-iot/agile-sdk/issues/new), we'll be happy to help.
   * @param {string} - agile-core REST API endpoint
+  * @param {string} - agile-idm REST API endpoint
+  * @param {string} - agile-idm token
   * @returns {Object}
   * @example
-  * var agile = require('agile-sdk')('http://agile-core:8080')
+  * var agile = require('agile-sdk')('http://agile-core:8080','http://agile-core:3000','zIOycOqbEQh4ayw7lGAm9ILBIr')
 */
-const agileSDK = (base) => {
+const agileSDK = (base,idmBase, token) => {
```

Comment:
done. I also updated the readme. There are two commits addressing all your changes